src/components/component.ts

```diff
@@ -33,18 +32,18 @@ module Plottable {
     private _usedLastLayout = false;
 
     /**
-     * Attaches the Component as a child of a given a DOM element. Usually only directly invoked on root-level Components.
+     * Attaches the Component as a child of a given a D3 selection.
```

Comment:
Funky grammar here. Also we should probably standardize references to "D3 selection" or "Selection" and just put "D3.Selection" every time so there's zero ambiguity about what sort of selection it is.